core/integration/features/verified/verified-positions-resolution-1.feature

```diff
@@ -87,7 +87,8 @@ Feature: Position resolution case 1
       | sellSideProvider | BTC   | ETH/DEC19 | 2154700 | 999997854000 |
       | buySideProvider  | BTC   | ETH/DEC19 | 320     | 999999999680 |
       | aux              | BTC   | ETH/DEC19 | 320     | 999999999650 |
-      | aux2             | BTC   | ETH/DEC19 | 7430    | 999999992600 |
+      | aux2             | BTC   | ETH/DEC19 | 7520    | 999999992510 |
+      #| aux2             | BTC   | ETH/DEC19 | 7430    | 999999992600 |
```

Comment:
can we not keep this historical values please? (commenting here for all of the files above)

Reply:
We sure can, I just kept them here to make reviewing/documenting things easier for @Jiajia-Cui and @witgaw